src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -76,7 +76,8 @@ public ApplicationInsightsLogger(TelemetryClient client, string categoryName, Ap
             {
                 [LogConstants.CategoryNameKey] = _categoryName,
                 [LogConstants.LogLevelKey] = (LogLevel?)logLevel,
-                [LogConstants.EventIdKey] = eventId.Id
+                [LogConstants.EventIdKey] = eventId.Id,
+                [LogConstants.EventName] = eventId.Name
```

Comment:
Can you also add `LogConstants.EventNameKey` to the `SystemScopeKeys` at the top of this class? That will prevent this from being double-logged.

Reply:
I think i can abondon this PR, as I @cgillum has already raised a simialr PR, @cgillum  can you please take care of these comments in your PR https://github.com/Azure/azure-webjobs-sdk/pull/2544/files

Reply:
@moolta will do. Thanks for the quick response!